lib/media/drm_engine.js

```diff
@@ -1975,7 +2023,8 @@ shaka.media.DrmEngine = class {
 
     // TODO: refactor, don't stick drmInfos onto MediaKeySystemConfiguration
     shaka.media.DrmEngine.processDrmInfos_(
-        config['drmInfos'], licenseServers, serverCerts, initDatas, keyIds);
+        config['drmInfos'], licenseServers, serverCerts,
+        serverCertificateUris, initDatas, keyIds);
 
     if (serverCerts.length > 1) {
```

Comment:
You should update this warning for the URIs too.  This should also warn that the buffer takes precedence over the URI.